WEB/Src/PerformanceCollector/PerformanceCollector/Implementation/Service contract/CollectionConfigurationInfo.cs

```diff
@@ -12,6 +12,15 @@ internal class CollectionConfigurationInfo
         [DataMember]
         public string ETag { get; set; }
 
+        [DataMember(IsRequired = false)]
```

Comment:
New DTO here:
- InitialQuota is optional. If it's set, we hard-reset _current_ quota when creating the new configuration. If it's not set, we use _current_ quota from the previous state of things to use as _initial_ for the new configuration. If this is the first configuration, then use a default.
- Max and AccrualRate are mandatory for the new DTO.
- The new DTO is optional. If the new DTO is not specified, then all values fall back to the default.